web/src/features/panels/zone/EstimationCard.cy.tsx

```diff
@@ -22,7 +22,7 @@ describe('EstimatedTSACard', () => {
     cy.get('[data-testid=title]').contains('Data is preliminary');
     cy.get('[data-testid="collapse-button"]').click();
     cy.get('[data-testid="body-text"]').contains(
-      'The data for this hour has not been reported yet and is based on preliminary data.'
+      "The data for this hour has not yet been reported. Our models fill gaps that data providers can't, providing a complete, granular dataset in real time."
```

Comment:
I might leave out the "gaps that data provider's can't" part.

Reply:
Rationale: it's not totally clear what data providers means in this case. I think it's strong enough without that wording as well.

Reply:
It's also a bit inaccurate as we sometimes remove values in the outlier detection pipeline because they are clearly wrong, but that don't mean it's a gap.

Reply:
I've made the small wording change. Will continue to improve this in the next PR to update the chart labelling.